UnitsNet/BaseDimensions.cs

```diff
@@ -151,5 +151,19 @@ private static void AppendDimensionString( StringBuilder sb, string name, int va
                     sb.AppendFormat( "^{0}", value );
             }
         }
+
+        public int Length { get { return _length; } }
```

Comment:
In C#7 (or was it C#6?) you no longer need this field. You can simply have a property `public int Mass { get; }` that is then assigned its value in the constructor.